src/analytics/index.ts

```diff
@@ -1,4 +1,6 @@
-import { Identify, identify, init, track } from '@amplitude/analytics-browser'
+import { add, Identify, identify, init, track } from '@amplitude/analytics-browser'
+import client from '@amplitude/analytics-browser'
```

Comment:
I think this should fix it

```suggestion
import * as client from '@amplitude/analytics-browser'
```

Reply:
Hi Kevin, appreciate your help. This is the error I'm getting now: 
<img width="841" alt="Screen Shot 2022-10-27 at 3 28 17 PM" src="https://user-images.githubusercontent.com/41491154/198381145-64e86310-d90a-4451-a16f-c2a8e9e91fbd.png">

I think the way I'm importing webAttributionPlugin might be incorrect? 
`import { webAttributionPlugin } from '@amplitude/plugin-web-attribution-browser'
`

Reply:
Ah, your code seems correct. I remember this issue which we have fixed in `@amplitude/analytics-browser@1.3.0`, I see your `yarn.lock` says you have `v1.1.4`. I'd suggest to bump to the latest and greatest 😄

Reply:
Hi Kevin, sorry for the late reply. Same error even when we update to the latest version 1.4.0. 

The ultimate goal is just to get our referral params again. 

Would you mind taking a look at our function "initializeAnalytics" in src/analytics/index.ts ? I believe this is where we set referral params to be included. Is our configuration missing anything that would cause referral params to be missing?

Reply:
Unfortunately, I'm not able to reproduce the error `Type 'void' is not assignable to type 'undefined'` in the newer versions.

The configuration looks fine. It's a mystery to me why the campaign parameters are not being captured after an update. If you are testing locally, you can try to lower the configuration `sessionTimeout` (perhaps 1000s) to force the session to expire and re-capture the campaign parameters on init. Reminder that for non-plugin solution, campaign parameters are only captured on new session, and optionally on campaign parameter change (configuration `attribution.trackNewCampaigns`), not necessarily on every page load.